### ISSUE TITLE
Contact Section and Footer

### ISSUE THREAD

**Issue by surasnayak** [surasnayak/clickies#8]
Need to add a contact section at the bottom of the page and also a footer section.

**Comment by ghost** [comment]
How do you want it to look like? Can I contribute to that?

**Comment by surasnayak** [comment]
@vireakpanha I am assigning you this for now. Get back to me with any doubts.

**Comment by ghost** [comment]
So, how do you want me to design it? The same as my previous design and change to REACT? or what?

**Comment by surasnayak** [comment]
Yes. Checkout the app.js file in the project and you will understand how things are done. And while making a pull request also add a screenshot of the application running on your local computer so that I can see the visualize and changes before merging.

**Comment by ghost** [comment]
How do you want your footer to look like actually?

**Comment by surasnayak** [comment]
There needs to be a contacts section having Name, Ph.No, email, Location. And social icons that can link to the profiles. Something like this.  View on map is not necessary.

P.S : Add some random values and links I will change them later.

![screenshot 61](https://user-images.githubusercontent.com/21239888/52961496-15441980-33c1-11e9-82ba-3e7b8e816041.png)

**Comment by ghost** [comment]
![screenshot_1](https://user-images.githubusercontent.com/45569747/52989442-fdf34380-3435-11e9-9e44-5bd78dc60dbc.png)
You want your footer to look like this?  Can you provide me with your detail?

**Comment by surasnayak** [comment]
Yes, but the background color should match that of the website and the social icons I need them to be white instead of black. The contact section should look like this.

< insert name here >
Ph.No : < insert phone number >
email : < insert email here >
Address : < insert address >

Make this layout with some random names and links. I will update my details myself later.

**Comment by ghost** [comment]
Okay! I'll get back to you as soon as possible.
